tools/chrome-debug-extension/package.json

```diff
@@ -43,8 +43,8 @@
     "license": "MIT",
     "sideEffects": false,
     "dependencies": {
```

Comment:
The chrome extension package.json version is unchanged, only changed dependencies version